client/my-sites/posts/post.jsx

```diff
@@ -368,6 +369,13 @@ module.exports = React.createClass({
 		return this.props.sites.getSite( this.props.post.site_ID );
 	},
 
+	onCommentsClick: function() {
+		this.setState( {
+			showComments: ! this.state.showComments
+		} );
+		this.analyticsEvents.commentIconClick();
```

Comment:
What do you think about renaming this function to `toggleComments`?

Just wondering if:
* we should also pass current state to `commentIconClick` method to have better idea how users use this feature?
* we should also add analytics for `toggleMoreControls` action?
:)